modular_newhorizons/master_files/code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -0,0 +1,147 @@
+
+/singleton/reagent/fluvectionem
+	name = "Calomel"
+	description = "A highly toxic medicine that quickly purges most chemicals from the bloodstream. Overdose causes bloodloss and more toxin buildup, however works twice as fast."
+
+/singleton/reagent/pulmodeiectionem
+	description = "A powdery chemical that damages the mucus lining in the the main broncus and the trachea, allowing particles to easily escape the lungs. Only works when inhaled. May cause long term damage to the lungs, and oxygen deprevation."
+
+/singleton/reagent/pneumalin
+	description = "A chemical that, when inhaled, restores tearing and bruising of the lungs. Overdosing can lower a patient's pulse to dangerous levels."
+
+/singleton/reagent/rezadone
+	description = "A powder with almost magical properties, this substance can effectively treat genetic damage in humanoids, though excessive consumption has side effects."
+
+/singleton/reagent/verunol
+	name = "Ipecac"
+	description = "A simple emetic, Induces vomiting in the patient, emptying stomach contents"
+
+/singleton/reagent/adipemcina
+	description = "Adipemcina is a heart medication used for treating high blood pressure, heart failure, and diabetes. Causes vomiting and liver damage when overdosed."
+
+/singleton/reagent/pacifier
+	description = "A mind altering chemical compound capable of suppressing violent tendencies."
+
+/singleton/reagent/rmt
+	description = "A chemical rampantly used by those seeking to remedy the effects of prolonged zero-gravity adaptations."
+
+/* MENTAL */
+
+/singleton/reagent/mental/corophenidate
+	name = "Methylphenidate"
+	description = "Methylphenidate is an ADHD treatment drug that treats basic distractions such as phobias and hallucinations at moderate doses. Withdrawl effects are rare. Side effects are rare, and include hallucinations."
+
+/singleton/reagent/mental/neurostabin
+	name = "Fluvoxamine"
+	description = "Fluvoxamine is safe and effective at treating basic phobias, as well as schizophrenia and muscle weakness at higher doses. Withdrawl effects are rare. Side effects are rare, and include hallucinations."
+
+/singleton/reagent/mental/parvosil
+	name = "Sertraline"
+	description = "Sertraline is cheap, safe, and effective at treating basic phobias, however it does not last as long as other drugs of it's class. Withdrawl effects are uncommon. Side effects are rare."
+
+/singleton/reagent/mental/minaphobin
+	name = "Escitalopram"
+	description = "Escitalopram is expensive, safe and very effective at treating basic phobias as well as advanced phobias like monophobia. A common side effect is drowsiness, and a rare side effect is hallucinations. Withdrawl effects are uncommon."
+
+/singleton/reagent/mental/emoxanyl
+	name = "Duloxetine"
+	description = "Duloxetine is effective at treating basic phobias and concussions. A rare side effect is hallucinations. Withdrawl effects are common."
```

Comment:
"A novel antioxidant medication which increases cerebral circulation and is used to treat basic phobias, anxiety, depression, concussion, and epilepsy. It has fewer side effects than many other forms of psychoactive drugs. Withdrawal symptoms are frequent and unsafe, and include hallucinations and heightened anxiety."